applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -559,30 +559,59 @@ To connect with an external MCU using UART_2, change the configuration files for
              pinctrl-names = "default", "sleep";
           };
 
-The following table shows how to connect an nRF52 Series development kit to an nRF91 Series development kit to be able to communicate through UART:
-
-.. list-table::
-   :align: center
-   :header-rows: 1
-
-   * - nRF52 Series DK
-     - nRF91 Series DK
-   * - UART TX P0.6
-     - UART RX P0.11
-   * - UART RX P0.8
-     - UART TX P0.10
-   * - UART CTS P0.7
-     - UART RTS P0.12
-   * - UART RTS P0.5
-     - UART CTS P0.13
-   * - GPIO OUT P0.27
-     - GPIO IN P0.31
-   * - GPIO IN P0.26
-     - GPIO OUT P0.30
+The following table shows how to connect selected development kit to an nRF91 Series development kit to be able to communicate through UART:
+
+.. tabs::
+
+   .. group-tab:: nRF52 DK
+
+      .. list-table::
+         :align: center
+         :header-rows: 1
+
+         * - nRF52 Series DK
+           - nRF91 Series DK
+         * - UART TX P1.02
+           - UART RX P0.11
+         * - UART RX P1.01
+           - UART TX P0.10
+         * - UART CTS P1.06
+           - UART RTS P0.12
+         * - UART RTS P1.07
+           - UART CTS P0.13
+         * - GPIO OUT P0.27
+           - GPIO IN P0.31
+         * - GPIO IN P0.26
+           - GPIO OUT P0.30
+         * - GPIO GND
+           - GPIO GND
+
+   .. group-tab:: nRF53 DK
+
+      .. list-table::
+         :align: center
+         :header-rows: 1
+
+         * - nRF53 Series DK
+           - nRF91 Series DK
+         * - UART TX P1.04
+           - UART RX P0.11
+         * - UART RX P1.05
+           - UART TX P0.10
+         * - UART CTS P1.06
+           - UART RTS P0.12
+         * - UART RTS P1.07
+           - UART CTS P0.13
+         * - GPIO OUT P0.23
+           - GPIO IN P0.31
+         * - GPIO IN P0.28
+           - GPIO OUT P0.30
+         * - GPIO GND
+           - GPIO GND
 
 Use the following UART devices:
 
-* nRF52840 or nRF52832 - UART0
+* nRF52 or nRF53 - UART0
 * nRF9160 or nRF91x1 - UART2
```

Comment:
```suggestion
* nRF52 or nRF53 Series devices - UART0
* nRF9160 or nRF91x1 SoCs - UART2
```

Reply:
I have put "Series DK" to both lines now. I do not know if this will work with Thingy's etc. At least Thingy91(x) doesn't work out of the box but requires additional work, I guess. So hence I'm not covering it but only DKs.